src/main/java/client/client.java

```diff
@@ -0,0 +1,78 @@
+package client;
+
+/**
+ * Created by Ксения on 12.12.2016.
+ */
+
+
+        import org.json.JSONArray;
+        import org.json.JSONException;
+        import org.json.JSONObject;
+
+        import java.io.*;
+        import java.net.Socket;
+        import java.util.Random;
+
+public class client {
+    public static void main(String[] args) throws IOException, JSONException {

```

Comment:
&#x1F34E; **Issue**
I suggest you to pull the main function out of the server and make a server an object rather than a bunch of static methods (which will make it more OOP that Procedural style).